build.rs

```diff
@@ -198,6 +198,7 @@ fn build_lib(compiled_libraries: &mut HashSet<Libs>, target: &String, lib: Libs)
                 "darwin",
                 &[
                     "gfx/angle/checkout/src/common/system_utils_mac.cpp",
+                    "gfx/angle/checkout/src/common/system_utils_apple.cpp",
```

Comment:
Looks like with new update of `mozangle`; `GetExecutablePath` was moved to `system_utils_apple.cpp`, so I had to add this file here.

Other than this, I have followed pretty much same steps we follow for building windows dll for `egl` and such. Also tested this version locally with servo. Works fine.

cc @sagudev

Reply:
This looks right according to: https://github.com/servo/mozangle/blob/be871c1f5413a105d43390f7a7362b07e88abb50/gfx/angle/targets/angle_common/moz.build#L172